synapse/config/tracer.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 New Vector Ltd
```

Comment:
these days it should be:

Copyright 2019 The Matrix.org Foundation C.I.C.

Reply:
right, I just copied and pasted.